test/system/src/main/java/io/pravega/test/system/framework/services/PravegaSegmentStoreService.java

```diff
@@ -127,20 +127,22 @@ private App createPravegaSegmentStoreApp() {
         //set env
         String zk = zkUri.getHost() + ":" + ZKSERVICE_ZKPORT;
 
-        //System properties to configure SS service.
-        String hostSystemProperties = setSystemProperty("pravegaservice.zkURL", zk) +
-                setSystemProperty("bookkeeper.zkAddress", zk) +
-                setSystemProperty("hdfs.hdfsUrl", "hdfs.marathon.containerip.dcos.thisdcos.directory:8020") +
-                setSystemProperty("autoScale.muteInSeconds", "120") +
+        //Environment variables to configure SS service.
+        Map<String, String> map = new HashMap<>();
+        map.put("ZK_URL", zk);
+        map.put("BK_ZK_URL", zk);
+        map.put("HDFS_URL", "hdfs.marathon.containerip.dcos.thisdcos.directory:8020");
+        map.put("CONTROLLER_URI", conUri.toString());
```

Comment:
@Prabha-Veerubhotla , good catch. We should change our variable to `CONTROLLER_URL`. This variable is set by the deployment script and if we change in `entrypoint.sh`, it is a breaking change for them. Updating `PravegaSegmentStoreService` to set `CONTROLLER_URL`.